com.unity.multiplayer.mlapi/Tests/Runtime/MultiInstanceHelpers.cs

```diff
@@ -104,15 +107,15 @@ public static void StopOneClient(NetworkManager clientToStop)
         /// </summary>
         public static void Destroy()
         {
-            if (!s_IsStarted)
+            if (s_IsStarted == false)
             {
-                throw new InvalidOperationException("MultiInstanceHelper is not started");
```

Comment:
@TwoTenPvP I think you added this a while ago. I'm removing it again to allow for save TearDown in case multi instance helpers are created inside a test and not during Setup